java-manta-client-unshaded/src/main/java/com/joyent/manta/util/MantaUtils.java

```diff
@@ -178,6 +182,43 @@ public static Boolean parseBooleanOrNull(final Object value) {
         return parsed;
     }
 
+    /**
+     * Parses an arbitrary object for a character. If it can't be found, return null.
+     *
+     * @param value Object to parse for a character
+     * @return if parsing fails, return null
+     */
+    public static Character parseCharOrNull(final Object value) {
```

Comment:
This method is unused. Please delete it. If you do end up using it, it is a really odd implementation and it would be good to discuss why you are doing it.

Reply:
It was present to potentially check for `delimiter` constraints in pagination support for buckets but it's no longer needed since validation will be taken care of in muskie (webapi).